pkg/tsdb/mssql/mssql_test.go

```diff
@@ -1263,6 +1263,50 @@ func TestMSSQL(t *testing.T) {
 			})
 		})
 	})
+
+	t.Run("Given an empty table", func(t *testing.T) {
+		type emptyObj struct {
+			EmptyKey string
+			EmptyVal int64
+		}
+
+		exists, err := sess.IsTableExist(emptyObj{})
+		require.NoError(t, err)
+		if exists {
+			err := sess.DropTable(emptyObj{})
+			require.NoError(t, err)
+		}
+		err = sess.CreateTable(emptyObj{})
+		require.NoError(t, err)
+
+		t.Run("When no rows are returned, should return an empty frame", func(t *testing.T) {
+			query := &backend.QueryDataRequest{
+				Queries: []backend.DataQuery{
+					{
+						JSON: []byte(`{
+							"rawSql": "SELECT empty_key, empty_val FROM empty",
```

Comment:
Heads up - this should be `FROM empty_obj` to match the struct name change. Same with `postgres_test.go` if you wanted to make that `emptyObj` as well.